### DISCUSSION TITLE
Just want to say Hi and ALSO this looks like an interesting project!

### DISCUSSION THREAD

**Discussion by rxt299299** [drkostas/Minecraft-AI#14] [General]
It let me remind the 3 years I spent building my own town in Minecraft!

**Comment by HsiangNianian** [comment]
That's an interesting idea. 💡
I have played Minecraft for more than six years, and I came into contact with programming and music because of it, and I once had this idea: in mc access AI, I am grateful that you have achieved a better solution!
